docs/content/concepts/transactions/prog-txn-blocks.mdx

```diff
@@ -7,28 +7,228 @@ In basic terms, a blockchain is a block of transactions chained together in a sh
 
 ## Transactions on Sui
 
-In Sui, a transaction is more than a basic record of the flow of assets. Transactions in Sui are composed of a number of commands that execute on inputs to define the result of the transaction. In essence, transactions on Sui are programmable, therefore they are termed "programmable transaction blocks" (PTB for short) to reference their additional capabilities over traditional blockchain transactions. A programmable transaction block on Sui can perform up to 1,024 unique operations in a single execution, whereas transactions on traditional blockchains would require 1,024 individual executions to accomplish the same result.
+On Sui, a transaction is more than a basic record of the flow of assets. Transactions on Sui are composed of a number of commands that execute on inputs to define the result of the transaction. Termed programmable transaction blocks (PTBs), these groups of commands define all user transactions on Sui. PTBs allow a user to call multiple Move functions, manage their objects, and manage their coins in a single transaction--without publishing a new Move package. Designed with automation and transaction builders in mind, PTBs are a lightweight and flexible way of generating transactions. More intricate programming patterns, such as loops, are not supported, however, and in those cases you must publish a new Move package.
 
-:::info
+As mentioned, each PTB is comprised of individual transaction commands (sometimes referred to themselves as transactions). Each transaction command executes in order, and you can use the results from one transaction command in any subsequent transaction command. The effects, specifically object modifications or transfers, of all transaction commands in a block are applied atomically at the end of the transaction. If one transaction command fails, the entire block fails and no effects from the commands are applied.
```

Comment:
```suggestion
As mentioned, each PTB is comprised of individual transaction commands (sometimes referred to themselves as transactions or simply commands). Each command executes in order, and you can use the results from one command in any subsequent transaction command. The effects, specifically object modifications or transfers, of all commands in a block are applied atomically at the end of the transaction. If one command fails, the entire block fails and no effects from the commands are applied.
```

For consistency with the other document (also, arguably reads a bit better without that many "transactions").

Reading further, though, I see that this document favors "transaction commands" over "commands". Perhaps I am too picky about it, but I feel like it would be better to be consistent in this.